test/framework/src/test/java/org/elasticsearch/test/hamcrest/ElasticsearchAssertionsTests.java

```diff
@@ -188,4 +200,49 @@ public void testAssertXContentEquivalentErrors() throws IOException {
             assertThat(error.getMessage(), containsString("expected [1] more entries"));
         }
     }
+
+    public void testAssertBlocked() {
+        {
```

Comment:
It looks like the four variants below do the same except for the set of cluster blocks. I would prefer to refactor this to share the setup and validation in a method and then call this with different cluster blocks.

Reply:
I have done that.